articles/private-saas-deployment/managed-private-cloud/index.md

```diff
@@ -0,0 +1,49 @@
+---
+section: private-cloud
+description: Overview of the Managed Private Cloud deployment option
+topics: managed-private-cloud
+contentType: concept
+useCase: private-cloud
+---
+# Managed Private Cloud
+
+Auth0's **Managed Private Cloud** (MPC) offering is a specially customized Auth0 deployment that allows you greater flexibility and increased input when it comes to day-to-day operations. As an MPC customer, you will be more involved with all aspects of your Auth0 deployment.
```

Comment:
Auth0's **Managed Private Cloud** (MPC) is a specially customized Auth0 deployment that gives you greater flexibility and increased input when it comes to day-to-day operations.